lars-reimann/Safe-DS
# feat: added Column.count_null_values() and test_count_null_values.py

Closes #160.

### Summary of Changes

Added count_null_values() method to Column class that returns an int with the number of null values. Also added validation test.

### Testing Instructions

Create a column with n 'null values'. Call the method count_null_values() and check if its n.
